### DISCUSSION TITLE
Error cannot access Web UI : Forbidden You don't have permission to access this resource.Server unable to read htaccess file, denying access to be safe

### DISCUSSION THREAD

**Discussion by IOTech17** [nextcloud/all-in-one#4039] [Questions] [answered]
Hi,

I am trying to deploy NextCloud AIO 7.9.1 via docker

I am using Nginx Proxy Manager on another docker server to route all the traffic via internet -> Cloudflare -> npm -> nextcloud

In my docker compose this is what I have setup :

       - APACHE_PORT=11000 
       - SKIP_DOMAIN_VALIDATION=true
       - APACHE_IP_BINDING=0.0.0.0
       - 
However I get this error message 

Forbidden
You don't have permission to access this resource.Server unable to read htaccess file, denying access to be safe

![image](https://github.com/nextcloud/all-in-one/assets/68947790/51ddf1a8-9a35-42cd-a454-74164b248891)

In the apache container logs I can see this : /var/www/html/.htaccess pcfg_openfile: unable to check htaccess file, ensure it is readable and that '/var/www/html/' is executable

### Steps to reproduce
Deploy Nextcloud AIO via docker -> follow the setup via the master container web console -> container are deployed -> try to login via the domain name -> get this error

### Host OS

FedoraCore OS latest

#### Nextcloud AIO version 

28

#### Current channel

Latest

**Comment by szaimen** [comment]
Sorry, misread. Are the permissions et correctly for the htaccess file?

**Reply by IOTech17** [reply]
Hi @szaimen 

Thank you for your quick reply

I do not have access to the .htaccess file as generated by the Apache container and I can't access via the bash from the container, the user doesn't have permission to access the file.

**Reply by szaimen** [reply]
Maybe it is a selinux issue? Can you check if disabling that makes it work?

**Reply by IOTech17** [reply]
I already uncommented this line in the docker-compose  (I am deploying via portainer) : 

security_opt: ["label:disable"]

**Reply by szaimen** [reply]
Yeah but this is not applied to the sibling containers. So can you for a test disable selinux and check if that makes it work?

**Reply by IOTech17** [reply]
So indeed setting up selinux to permissive and reooting the docker host solved the problem.

I can see that in the logs : type=AVC msg=audit(1704976389.369:799): avc:  denied  { read } for  pid=11176 comm="httpd" name=".htaccess" dev="vda4" ino=239369404 scontext=system_u:system_r:container_t:s0:c840,c863 tcontext=system_u:object_r:container_file_t:s0:c219,c433 tclass=file permissive=1


How to whitelist this behaviour in se linux?

I prefere having selinux enabled.

**Reply by IOTech17** [reply]
Would it be possible to pass on the selinux disable for the apache container in future release?

**Reply by szaimen** [reply, marked as answer]
> Would it be possible to pass on the selinux disable for the apache container in future release?

Yes, see https://github.com/nextcloud/all-in-one/pull/4047

**Reply by szaimen** [reply]
This is now released with v7.11.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Reply by IOTech17** [reply]
@szaimen I tested and confirm that it is working!

thank you for your quick support!
